### ISSUE TITLE
Problem when starting application on windows

### ISSUE THREAD

**Issue by luisfpinto** [netbeast/dashboard#93]
```
child_process.js:1162
   throw errnoException(err, 'spawn');
         ^
Error: spawn UNKNOWN
   at exports._errnoException (util.js:746:11)
   at ChildProcess.spawn (child_process.js:1162:11)
   at exports.spawn (child_process.js:995:9)
   at d:\netBeast Github\dashboard\src\models\activity.js:117:17
   at d:\netBeast Github\dashboard\src\models\app.js:75:5
   at d:\netBeast Github\dashboard\node_modules\fs-extra\node_modules\jsonfile\index.js:39:5
   at fs.js:334:14
   at d:\netBeast Github\dashboard\node_modules\decompress\node_modules\vinyl-fs\node_modules\graceful-fs\graceful-fs.js:43:10
   at d:\netBeast Github\dashboard\node_modules\gulp-sourcemaps\node_modules\graceful-fs\graceful-fs.js:43:10
   at d:\netBeast Github\dashboard\node_modules\fs-extra\node_modules\graceful-fs\graceful-fs.js:43:10
   at d:\netBeast Github\dashboard\node_modules\mosca\node_modules\st\node_modules\graceful-fs\graceful-fs.js:43:10
```

**Comment by luisfpinto** [comment]
How to fix: 

File: dashboard/src/models/activity.js
Line 117
```
var child = spawn('node', [entryPoint, '--port', app.port], {
      cwd: path.join(APPS_DIR, app.name),
      env: env
    })
```

**Comment by luisfpinto** [comment]
@Mo33n Please close this issue when you create the PR :smile:

**Comment by Mo33n** [comment]
Bug solved #96
